index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main>
+        <nav class="nav-bar">
+            <ul>
+                <li><a href="#"> <img class="apple-img" src="images/apple.png"> </a></li>
+                <li><a href="#"> Store </a></li>
+                <li><a href="#"> Mac </a></li>
+                <li><a href="#"> iPod </a></li>
+                <li><a href="#"> iPhone </a></li>
+                <li><a href="#"> iPad </a></li>
+                <li><a href="#"> iTunes </a></li>
+                <li><a href="#"> Support </a></li>
+                <li><img class="search-icon" src="images/searchicon.png"> <input type="text"></li>
+            </ul>
+        </nav>
+        <section class="main-header">
+            <a href="#">
+                <div class="background-image hero">
+                    <img class="img-title" src="images/your_verse_title_small.png"> 
+                    <img class="img-text" src="images/your_verse_subhead_small.png">
+                    <div class="box-image">
+                        <img src="images/cta_your_verse_27pt.png">
+                    </div>
```

Comment:
The images in the div should  be passed as background images and you should have three links, that is each images should be held by a div tag that is wrapped in an `<a>` tag

Reply:
Ok.. I have three links now, with a div inside. Each div have background images.